canonical-web-and-design/ubuntu.com
# Select the most recently started subscription

## Done

- Select the most recently started subscription by default.

## QA

- Visit [/advantage?test_backend=true](/advantage?test_backend=true) and log in.
- The subscription at the top of the list should be selected on first load.


## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/101.
